packages/flutter/lib/src/widgets/primary_scroll_controller.dart

```diff
@@ -34,14 +44,18 @@ class PrimaryScrollController extends InheritedWidget {
   const PrimaryScrollController({
     super.key,
     required ScrollController this.controller,
+    this.autoForPlatforms = _kMobilePlatforms,
```

Comment:
I do not like this name, but have not thought of something else yet.

Reply:
It is tricky because it **can** be inherited on any platform, this is specific to the automatic inheritance.

Reply:
> can you do anything with this of running on a platform that's not part of this set?

I am not sure I understand this question